### ISSUE TITLE
Rmarkdown cannot plot 

### ISSUE THREAD

**Issue by AntPalatov** [rstudio/rmarkdown#916]
Hi, I discovered an issue. When I open new Rmarkdown file it generates example with summary(cars) and plot(pressure).  It shows summary table but raises an error for plot.  
The same plot for pressure I can print from console or can see in the knitted  pdf/html file
Rstudio version 1.0.44

> Error in (function (filename = "Rplot%03d.png", width = 480, height = 480, : cannot start  png()
> 10. (function (filename = "Rplot%03d.png", width = 480, height = 480, units = "px", pointsize = 12, bg = "white", res = NA, family = "sans", restoreConsole = TRUE, type = c("windows", "cairo", "cairo-png"), antialias = c("default", "none", "cleartype", "gray", "subpixel")) ...
> 9. do.call(what = png, args = args)
> 8.rs.createNotebookGraphicsDevice(filename, height, width, units, pixelRatio, extraArgs)
> 7 (function () { .rs.createNotebookGraphicsDevice(filename, height, width, units, pixelRatio, extraArgs) ...
> 6 dev.hold()
> 5 plot.default(x[[1L]], x[[2L]], xlab = xlab, ylab = ylab, ...)
> 4 plot(x[[1L]], x[[2L]], xlab = xlab, ylab = ylab, ...)
> 3 plot2(x, ...)
> 2 plot.data.frame(pressure)
> 1 plot(pressure)

```
> sessionInfo()
R version 3.3.2 (2016-10-31)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

locale:
[1] LC_COLLATE=Russian_Russia.1251  LC_CTYPE=Russian_Russia.1251    LC_MONETARY=Russian_Russia.1251
[4] LC_NUMERIC=C                    LC_TIME=Russian_Russia.1251    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
 [1] backports_1.0.4 magrittr_1.5    rprojroot_1.1   htmltools_0.3.5 tools_3.3.2     yaml_2.1.14     Rcpp_0.12.8    
 [8] stringi_1.1.2   rmarkdown_1.2   knitr_1.15.1    stringr_1.1.0   digest_0.6.10   evaluate_0.10  
```

**Comment by AntPalatov** [comment]
issue closed with new version of Rstudio 1.0.136

**Comment by yihui** [comment]
Thanks for the follow-up!
